### PR TITLE
i2073: Issue cookie on login

### PR DESCRIPTION
**Pull request by MartinEden**
https://vimc.myjetbrains.com/youtrack/issue/VIMC-2073

### CODE REVIEW

File: src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/UserLogicTests.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    fun `getUserByUsername passes through to repository`()
+    {
+        val repo = mock<UserRepository>()
+        val sut = RepositoriesUserLogic(repo, mock())
+        sut.getUserByUsername("user")
+        verify(repo).getUserByUsername("user")
+    }
+
+    @Test
+    fun `logInAndGetToken gets token and updates last logged in`()
```
Comment (littledice) [inline src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/UserLogicTests.kt]: Looks like we had a test before that checked that `lastLoggedIn` wasn't updated on a failed login, but that's no longer under test? Unless I've missed it

Comment (MartinEden) [reply src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/UserLogicTests.kt]: This is still tested, just less directly: In the controller test, we check that the logic method `logInAndGetToken` is not called. And in the logic test we check that calling `logInAndGetToken` sets `lastLoggedIn`.

Comment (littledice) [reply src/app/src/test/kotlin/org/vaccineimpact/api/tests/logic/UserLogicTests.kt]: Ah yes, ok

**Comment by MartinEden** [comment]
Make sure we return webmodels to master after merging https://github.com/vimc/montagu-webmodels/pull/62

**Pull request merged**